packages/now-cli/test/integration.js

```diff
@@ -119,6 +119,7 @@ const defaultArgs = [];
 let token;
 let email;
 let contextName;
+let userId;
```

Comment:
I'd rather not add more global variables to the tests, use `fetchTokenInformation` or `apiFetch` instead.

Reason is that every test should work on it's own, and we use `now login` in one to change the user and I'm planning on switching users more often, since we run into deployment rate limits a lot for those tests.

Reply:
Tackled in 1128b8931cffc0717246f47f9ddcf1e77e3a4aa2